### ISSUE TITLE
MS Binding vs Manual Binding - Possibly a bug

### ISSUE THREAD

**Issue by nikdejan** [YetiForceCompany/YetiForceCRM#1621]
With normal scheduled binding everything works correctly. Emails I send from Opportunity (by clicking Send Email button in Email widget) are categorized as Sent in Corporate Email and in widget. 
BUT, if I send an email and then go to Roundcube's Sent folder BEFORE MailScanner gets executed, after opening an email I just sent and refreshing, it gets bind to Opportunity, but it is categorizes as Received, not as sent. 
So, basically, what ever gets bind that way (by simply opening email in mail client) is categorized as Received.
